docker/docker-compose.yml

```diff
@@ -42,12 +57,6 @@ services:
     restart: unless-stopped
     ports:
       - 33807:33807
-    healthcheck:
-      test: ['CMD', 'mysqladmin', 'ping', '-h127.0.0.1', '-P33577']
```

Comment:
This healthcheck was not good and always failed, see fixed version below.

Reply:
Using the correct port helps :hidethepain: